modules/tensor_mechanics/tests/generalized_plane_strain/tests

```diff
@@ -1,10 +1,26 @@
 [Tests]
```

Comment:
Please do add in the tests for all three different strain formulations so that we have testing code coverage for the new strain materials.  You could likely use the same test for the small strain and the small incremental strain, using command line arguments in the test file, so that you would only need to add two new tests.